LoginDemo/LearningSelenium/PageModel/Login.cs

```diff
@@ -0,0 +1,84 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace LearningSelenium.PageModel
+{
+    public class Login
+    {
+        WebDriver driver;
+        protected By UserName = By.XPath("//input[contains(@name,'username')]");
+        protected By Password = By.XPath("//input[contains(@name,'password')]");
+        protected By LogButton = By.XPath("//button[contains(@type,'submit') ]");
+        public By LogError = By.XPath("//p[contains(@class,'oxd-text') and contains(@class,'oxd-text--p') and contains(@class,'oxd-alert-content-text') ]");
```

Comment:
Remove extra spaces in locators

Reply:
extra spaces removed
No need to use all the classes if we have something common for other attribute in that case it require.